content/docs/setup/terraform/_index.md

```diff
@@ -6,52 +6,340 @@ weight = 30
+
+```
+tink-server_1  | {"level":"info","ts":1592936829.6773047,"caller":"grpc-server/workflow.go:63","msg":"done creating a new workflow","service":"github.com/tinkerbell/tink"}
+```
+
+## Check workflow status
+
+You can not ssh into the worker but you can use the CLI from the provisioner to
+validate if the workflow completed correctly using the `tink workflow events`
+command (an event to show up can take ~5 minutes, check the SOS console for the
+worker to see what it is doing):
+
+```
+$ docker exec -i deploy_tink-cli_1 tink workflow events $WORKFLOW_ID
+>
++--------------------------------------+-------------+-------------+----------------+---------------------------------+--------------------+
+| WORKER ID                            | TASK NAME   | ACTION NAME | EXECUTION TIME | MESSAGE                         |      ACTION STATUS |
++--------------------------------------+-------------+-------------+----------------+---------------------------------+--------------------+
+| ce2e62ed-826f-4485-a39f-a82bb74338e2 | hello world | hello_world |              0 | Started execution               | ACTION_IN_PROGRESS |
+| ce2e62ed-826f-4485-a39f-a82bb74338e2 | hello world | hello_world |              0 | Finished Execution Successfully |     ACTION_SUCCESS |
++--------------------------------------+-------------+-------------+----------------+---------------------------------+--------------------+
+```
+
+---
+
+The `SOS` or `Out of bond` console provided by Packet is a very nice way to
+follow what happens in the `worker` when the provisioner starts. My suggestion
+is to ssh in the SOS console and watch it as well. The `SOS` or `Out of bond`
+console provided by Packet is a very nice way to follow what happens in the
+`worker` when the provisioner starts. My suggestion is to ssh in the SOS console
+and watch it as well.
+
+You can the command to ssh in:
+
+```
+ssh $(terraform output -json worker_sos | jq -r '.[0]')
+```
+
+## Cleanup
+
+You can terminate worker and provisioner with the terraform destroy command:
+
+```
+$ terraform destroy
+```
+
+## FAQ
+
+> Error: The facility sjc1 has no provisionable c3.small.x86 servers matching
+> your criteria.
+
+This error notifies you that the facility you are using (by default sjc1) does
+not have devices available for `c3.small.x86`. You can change your device
+setting a different `device_type` in `terraform.tfvars` (be sure that layer2 is
+supported for the new device_type) or you can change facility with the variable
+`facility` set to a different one.
+
+In order to check availability of device type in a particular facility you can
+use the packet-cli:
+
+```
+$ packet capacity get
+```
+
+You are looking for a facility that has a `normal` lavel of `c3.small.x84`.
+
+> Error: timeout - last error: SSH authentication failed
+> (root@136.144.56.237:22): ssh: handshake failed: ssh: unable to authenticate,
+> attempted meth ods [none publickey], no supported methods remain
+
+We use the Terraform
```

Comment:
We should move these instructions to the beginning because the terraform provisioning requires this. We should also say that they should have their SSH key registered, "Add your key to the Packet Portal" https://www.packet.com/developers/docs/servers/key-features/ssh-keys/

Reply:
We will always get something "missing" or not detailed enough. We can also explain how to generate an SSH key or how to copy it. At some point, we just have to make some assumption I presume

Reply:
The need to set your ssh-agent is explained at the beginning of the documentation 👍

Reply:
I see the mention of the need for the SSH Agent in the beginning, but I think we should also emphasize that these keys must be registered in the Packet Portal's SSH Keys panel.

Reply:
Otherwise, Terraform will fail to provision and it will not be possible to login.